src/main/resources/default/assets/tycho/styles/tycho.scss

```diff
@@ -23,5 +23,6 @@
 
 @import "/assets/tycho/styles/sirius-colors";
 @import "/assets/tycho/styles/colors";
+@import "/assets/tycho/styles/scireum-core";
```

Comment:
do we really want this name here?

Reply:
We can extend this list via the config (core already injects additional scss as well as biz and the products)